npm/javy/README.md

```diff
@@ -0,0 +1,9 @@
+# Javy Lib
+
+This library provides abstractions and convenience methods over Javy's built-in functions.
+
+More documentation TBD>
+
+## Tests
+
+To run the tests, run `npm test`. It requires you to have built `javy` and have `wasmtime` installed.
```

Comment:
Maybe it's worth specifying that we should build Javy in release mode, since the release path is the one that is hardcoded.